synchronicity/utils.py

```diff
@@ -0,0 +1,32 @@
+import functools
+import os
+import traceback
+
+def should_skip_tb_frame(tb):
+    "Skip the frame if it's in the same directory as this module."
+
+    synchronicity_dir = os.path.dirname(__file__)
```

Comment:
not blocking but this could go in the global scope

Reply:
updated